tutorial/source/bayesian_regression.ipynb

```diff
@@ -123,7 +123,7 @@
     "    y_data = data[:, -1]\n",
     "    for j in range(num_iterations):\n",
     "        # run the model forward on the data\n",
-    "        y_pred = regression_model(x_data)\n",
+    "        y_pred = regression_model(x_data).squeeze(-1)\n",
```

Comment:
@jpchen - `y_pred` returns a tensor of size `100x1` on PyTorch master now, whereas `y_data` has size `100` and hence it errors when we compute the loss. Also, removed the other `squeeze` operations without which things seem to run fine.

Reply:
this looks fine, can you please change this in the example as well? and test with python3 also if you havent, since there was a squeeze bug in pytorch when doing `backward()`.